src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/SymbolMatcherTests.cs

```diff
@@ -1811,5 +1811,261 @@ static void Main()
             var field1 = (Cci.IFieldDefinition)matcher.MapDefinition(field0);
             Assert.Equal("<>9__0_0", field1.Name);
         }
+
+        [Fact]
+        public void CheckedUserDefinedOperators_01()
+        {
+            const string source =
+@"
+class A 
+{
+    public static A operator -(A c) => c;
+    public static A operator checked -(A c) => c;
+
+    public static A operator +(A c, A d) => c;
+    public static A operator checked +(A c, A d) => c;
+
+    public static explicit operator int(A c) => 0;
+    public static explicit operator checked int(A c) => 0;
+}
+";
+            var compilation0 = CreateCompilation(source, options: TestOptions.DebugDll);
+            var compilation1 = compilation0.WithSource(source);
+
+            var matcher = new CSharpSymbolMatcher(
+                null,
```

Comment:
Consider using named parameters for these `null`s, here and for other usages in this file.